QtCollider/primitives/prim_misc.cpp

```diff
@@ -167,7 +167,7 @@ QC_LANG_PRIMITIVE( Qt_SetGlobalPalette, 1, PyrSlot *r, PyrSlot *a, VMGlobals *g
 {
   if( !QcApplication::compareThread() ) return QtCollider::wrongThreadError();
 
-  QPalette p = QtCollider::get( a );
+  QPalette p = (QPalette&&) QtCollider::get(a);
```

Comment:
Unfortunately not:
C:\Users\Rainer\Projects\sc\supercollider\QtCollider\primitives\prim_misc.cpp(174): error C2440: 'static_cast' : cannot convert from 'QtCollider::DecodableSlot' to 'QPalette' [C:\Users\Rainer\Projects\sc\supercollider\build3.7VS64\lang\libsclang.vcxproj]